wifi: scan: Support for Raw scan (#11137)
nrfconnect/sdk-nrf: samples/wifi/scan/README.rst

To build for the nRF7002 DK, use the ``nrf7002dk_nrf5340_cpuapp`` build target.

> The following is an example of the CLI command:

Reviewer: ```suggestion
The following are examples of the CLI commands:
```
